### PR TITLE
Destruction is imminent

### PR DESCRIPTION
**Pull request by BrennanConroy**
Adding virtual destructor to web_request and web_request_stub to avoid memory leaks.

### CODE REVIEW

File: src/signalrclient/web_request.h [hunk 1]
```diff
@@ -22,6 +22,8 @@ namespace signalr
 
         web_request& operator=(const web_request&) = delete;
 
+		virtual ~web_request();
```
Comment (moozzyk) [inline src/signalrclient/web_request.h]: are you using tabs? if yes, then switch to spaces  (4 spaces)

File: test/signalrclienttests/web_request_stub.h [hunk 2]
```diff
@@ -25,4 +25,6 @@ struct web_request_stub : public web_request
     virtual void set_headers(const std::unordered_map<utility::string_t, utility::string_t>& headers) override;
 
     virtual pplx::task<web_response> get_response() override;
+
+	virtual ~web_request_stub();
```
Comment (moozzyk) [inline test/signalrclienttests/web_request_stub.h]: do we need it at all? also add `override;`

Comment (BrennanConroy) [inline test/signalrclienttests/web_request_stub.h]: You're right, we don't need this. Will remove.

Comment (moozzyk) [inline test/signalrclienttests/web_request_stub.h]: :+1:

**Comment by moozzyk** [comment]
:ship::it:

**Pull request closed without merging**